.buildkite/pipelines/build_linux.json.py

```diff
@@ -141,6 +144,15 @@ def main(args):
                         choices=actions,
                         default="build",
                         help="Specify a build action.")
+    parser.add_argument("--snapshot",
+                        required=False,
+                        choices=build_snapshot,
+                        default=None,
+                        help="Specify if a snapshot build is wanted.")
+    parser.add_argument("--candidate",
```

Comment:
I think `candidate` should be `version_qualifier` throughout this PR. Using different terminology to the release manager, other repos, and some other scripts in this repo is only going to cause confusion in the future.